cli/args.go

```diff
@@ -110,6 +110,12 @@ func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, wri
 		return nil, err
 	}
 
+	envValue, envProvided := os.LookupEnv("TERRAGRUNT_STS_DURATION")
+	stsDuration, err := parseIntArg(args, OPT_TERRAGRUNT_STS_DURATION, envValue, envProvided, 3600)
```

Comment:
NIT: extract `3600` into a constant with a clear name (e.g., `default_iam_role_duration`) and comment (e.g., `# one hour`).